tests/test_unittest/test_gtc/test_gtir.py

```diff
@@ -53,69 +41,18 @@
 A_ARITHMETIC_OPERATOR = ArithmeticOperator.ADD
 
 
-@pytest.fixture
-def copy_assign():
-    yield ParAssignStmt(
-        loc=SourceLocation(line=3, column=2, source="copy_gtir"),
-        left=FieldAccess.centered(
-            name="foo", loc=SourceLocation(line=3, column=1, source="copy_gtir")
-        ),
-        right=FieldAccess.centered(
-            name="bar", loc=SourceLocation(line=3, column=3, source="copy_gtir")
-        ),
-    )
-
-
-@pytest.fixture
-def interval(copy_assign):
-    yield Interval(
-        loc=SourceLocation(line=2, column=11, source="copy_gtir"),
-        start=AxisBound(level=LevelMarker.START, offset=0),
-        end=AxisBound(level=LevelMarker.END, offset=0),
-    )
-
-
-@pytest.fixture
-def copy_v_loop(copy_assign, interval):
-    yield VerticalLoop(
-        loc=SourceLocation(line=2, column=1, source="copy_gtir"),
-        loop_order=LoopOrder.FORWARD,
-        interval=interval,
-        body=[copy_assign],
-        temporaries=[],
-    )
-
-
-@pytest.fixture
-def copy_computation(copy_v_loop):
-    yield Stencil(
+def test_copy():
+    copy_computation = StencilFactory(
```

Comment:
I'm not sure it is a good idea to rely on the factory for this test. The whole purpose of the test is to test construction and some basic properties of the node, but by using the factory a test failure will occur inside the factory, making a fix rather inconvenient. I think the original test is fine, is there a reason to change it here?

Reply:
I agree that the error is poorly formatted when using the factories, but this is also really fragile because if any attribute changes, the tests break. I'm also not sure this is the use-case for pytest fixtures. I suppose I thought this was written before the factories were added.

That said, would you rather I switch it back? I'm OK doing that if you think the semantic of creating nodes without factories is critical.